android/automated_build/install-qt.sh

```diff
@@ -0,0 +1,339 @@
+VERSION=
+FORCE_DOWNLOAD=false
+MD5_TOOL=md5sum
+
+case "$OSTYPE" in
+    *linux*)
+        HOST_OS=linux_x64
+        INSTALL_DIR=/opt/Qt
+        TOOLCHAIN=gcc_64
+        ;;
+    *darwin*)
+        HOST_OS=mac_x64
+        INSTALL_DIR=/usr/local/Qt
+        TOOLCHAIN=clang_64
+        MD5_TOOL="md5 -r"
+        ;;
+    msys)
+        HOST_OS=windows_x86
+        INSTALL_DIR=/c/Qt
+        TOOLCHAIN=win64_msvc2015_64
+        ;;
+    *)
+        HOST_OS=
+        INSTALL_DIR=
+        ;;
+esac
+
+while [ $# -gt 0 ]; do
+    case "$1" in
+        --directory|-d)
+            INSTALL_DIR="$2"
+            shift
+            ;;
+        --force|-f)
+            FORCE_DOWNLOAD=true
+            ;;
+        --host)
+            HOST_OS="$2"
+            shift
+            ;;
+        --target)
+            TARGET_PLATFORM="$2"
+            shift
+            ;;
+        --toolchain)
+            TOOLCHAIN=$(echo $2 | tr '[A-Z]' '[a-z]')
+            shift
+            ;;
+        --version)
+            VERSION="$2"
+            shift
+            ;;
+        --help|-h)
+            help
+            exit 0
+            ;;
+        *)
+            COMPONENTS="${COMPONENTS} $1"
+            ;;
+    esac
+    shift
+done
+
+if [ -z "${HOST_OS}" ]; then
+    echo "No --host specified or auto-detection failed." >&2
+    exit 1
+fi
+
+if [ -z "${INSTALL_DIR}" ]; then
+    echo "No --directory specified or auto-detection failed." >&2
+    exit 1
+fi
+
+if [ -z "${VERSION}" ]; then
+    echo "No --version specified." >&2
+    exit 1
+fi
+
+if [ -z "${COMPONENTS}" ]; then
+    echo "No components specified." >&2
+    exit 1
+fi
+
+case "$TARGET_PLATFORM" in
+    android)
+        ;;
+    ios)
+        ;;
+    desktop)
+        ;;
+    *)
+        echo "Error: TARGET_PLATFORM=${TARGET_PLATFORM} is not valid." >&2
+        exit 1
+        ;;
+esac
+
+HASH=$(echo "${OSTYPE} ${TARGET_PLATFORM} ${TOOLCHAIN} ${VERSION} ${INSTALL_DIR}" | ${MD5_TOOL} | head -c 16)
+HASH_FILEPATH="${INSTALL_DIR}/${HASH}.manifest"
+INSTALLATION_IS_VALID=false
+if ! ${FORCE_DOWNLOAD} && [ -f "${HASH_FILEPATH}" ]; then
+    INSTALLATION_IS_VALID=true
+    while read filepath; do
+        if [ ! -e "${filepath}" ]; then
+            INSTALLATION_IS_VALID=false
+            break
+        fi
+    done <"${HASH_FILEPATH}"
+fi
+
+if ${INSTALLATION_IS_VALID}; then
+    echo "Already installed. Skipping download." >&2
+    exit 0
+fi
+
+DOWNLOAD_DIR=`mktemp -d 2>/dev/null || mktemp -d -t 'install-qt'`
+
+#
+# The repository structure is a mess. Try different URL variants
+#
+function compute_url(){
+    local COMPONENT=$1
+    local CURL="curl -s -L"
+    local BASE_URL="https://mirrors.ocf.berkeley.edu/qt/online/qtsdkrepository/${HOST_OS}/${TARGET_PLATFORM}"
+    local ANDROID_ARCH=$(echo ${TOOLCHAIN##android_})
+
+    if [[ "${COMPONENT}" =~ "qtcreator" ]]; then
+
+        SHORT_VERSION=${VERSION%??}
+        BASE_URL="https://mirrors.ocf.berkeley.edu/qt/official_releases/qtcreator"
```

Comment:
Same here

Reply:
will stick to updated version of install-qt